src/status_im/common/alert_banner/view.cljs

```diff
@@ -0,0 +1,57 @@
+(ns status-im.common.alert-banner.view
+  (:require [quo.core :as quo]
+            [quo.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [react-native.hole-view :as hole-view]
+            [react-native.safe-area :as safe-area]
+            [status-im.common.alert-banner.style :as style]
+            [utils.re-frame :as rf]))
+
+(def colors-map
+  {:alert
+   {:background-color colors/warning-50-opa-20
+    :text-color       colors/warning-60}
+   :error
+   {:background-color colors/danger-50-opa-20
+    :text-color       colors/danger-60}})
+
+(defn- banner
+  [{:keys [text type]} second-banner?]
+  [rn/view (when second-banner? {:style style/second-banner-wrapper})
+   [hole-view/hole-view
+    {:style (style/hole-view (get-in colors-map [type :background-color]))
+     :holes (if second-banner?
+              []
+              [{:x            0
+                :y            40
+                :width        (:width (rn/get-window))
+                :height       40
+                :borderRadius 20}])}
+    [quo/text
+     {:size   :paragraph-2
+      :weight :medium
+      :style  {:color (get-in colors-map [type :text-color])}}
+     text]]])
+
+(defn view
+  []
+  (let [banners       (rf/sub [:alert-banners])
+        banners-count (count banners)
+        alert-banner  (:alert banners)
+        error-banner  (:error banners)
+        safe-area-top (safe-area/get-top)]
+    [hole-view/hole-view
+     {:style {:background-color colors/neutral-100}
```

Comment:
Not sure we should set this color permanently. Maybe we should check how it looks in light mode.

Reply:
Thank you for review Javid, I discussed this with mario and he mentioned alert banner will always use dark theme.

Reply:
Great! 🙌

Reply:
In that case, this will affect the color of the status bar. Correct me if I'm wrong.

Reply:
> In that case, this will affect the color of the status bar. Correct me if I'm wrong.

Thank you very much for highlighting this. I will keep this in mind while working for status-bar color implementation for https://github.com/status-im/status-mobile/issues/19072